Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -67,44 +77,55 @@ class TransactionHistoryViewController: UIViewController {
             self.refreshControl.endRefreshing()
             return
         }
-        
+
         self.allTransactions = txs
+        self.applyTxSorter()
         self.transactionsTableView.backgroundView = nil
         self.transactionsTableView.separatorStyle = .singleLine
         self.setupTxFilterAndDisplayAllTxs()
     }
-    
+
+    func setupTxSorter() {
+        let sorterOptions = [LocalizedStrings.newest,
+                             LocalizedStrings.oldest]
+        self.transactionSorters = [TransactionSorterType.newest, TransactionSorterType.oldest]
+        self.transactionSorterDropDown.initMenu(sorterOptions) { [weak self] index, value in
+            self?.applyTxSorter()
+            self?.reloadTxsForCurrentFilter()
```

Comment:
As mentioned above, delegate sorting to dcrlibwallet. Instead of sorting here and reloading txs for current filter, you could just simply call `self.reloadTxsForCurrentFilter()`. Will need to modify that function so that whenever it is called, it takes into consideration the current filter.